README.md

```diff
@@ -87,6 +87,11 @@ $ boot environ -e database-url=jdbc:postgres://localhost/dev repl
 
 The latter form can be included in custom pipelines and `task-options!'.
 
+The task also creates or updates a `.boot-env` file in the fileset.
+This is useful for tasks that create their own pods like
+[`boot-test`](https://github.com/adzerk-oss/boot-test), which won't
```

Comment:
Oh, could you remove the backticks around boot-test while you're changing things? It's the name of a library rather than a section of code, so I don't think it needs code tags. If you don't get round to it, I'll change this in a commit of my own anyway.